### DISCUSSION TITLE
V9:  Is there an ETA on the Field Component?

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#27299] [Q&A] [answered]
Hi,

I was wondering when the Field component is expected to be stable for use. This will enable creating forms V9 and make it easier to migrate from V8

[https://react.fluentui.dev/?path=/docs/preview-components-field--default](https://react.fluentui.dev/?path=/docs/preview-components-field--default)

Thanks.

**Comment by behowell** [comment, marked as answer]
Hi @blendsdk, the Field component should be stable in early to mid-April. 

If you want to start with the unstable Field in the meantime, there is one breaking change coming, but it will only affect cases where Field is used with a control that's not part of the FluentUI library (like a third-party input control).

```jsx
<>
// ✅ All controls in the library will continue to work
<Field label="Example">
  <Input id="example" />
</Field>

// ⚠️ This will no longer work after the breaking change: controls that are not from the fluentui library
<Field label="Example">
  <ThirdPartyInputControl id="example" />
</Field>

// 💡 You'll need to pass a render function for non-fluentui controls after the breaking change. You can already do this today.
<Field label="Example">
   {fieldProps => <ThirdPartyInputControl {...fieldProps} id="example" />}
</Field>
</>
```

Let me know if you have any other questions. Thanks!

**Reply by blendsdk** [reply]
Thank you.

**Reply by behowell** [reply]
Hi @blendsdk, just following up that [Field](https://react.fluentui.dev/?path=/docs/components-field--default) is now stable in `@fluentui/react-components` v9.19.0.

**Reply by blendsdk** [reply]
Oh, cool thank you :)
